schema.graphql

```diff
@@ -327,10 +327,38 @@ enum FastUsdcTransactionStatus {
 
 type FastUsdcTransaction @entity {
```

Comment:
```FR1.4: MUST Enable search by transaction ID or addresses```
I think the `id` in this entity is the original EVM transaction ID right? That's what we want, so just making sure.

```
FR2.2: MUST Show all transaction timestamps:
- Source Chain Transaction Time
- LP Funds Disbursement
- CCTP Transaction Completion
- LP Settlement Time
```
Looks like `sourceBlockTimestamp` covers "Source Chain Transaction Time". Do we need to add timestamps to all the other status changes, or are they inferable somehow already?

Reply:
Also:
```
FR1.1: MUST Display paginated list of all Fast USDC transactions
```
It's reasonable to assume these have to be ordered in some way, like by `sourceBlockTimestamp`. It would be helpful to see an example query in the PR description to verify that it is possible after this change.

Reply:
>I think the id in this entity is the original EVM transaction ID right? That's what we want, so just making sure.

Correct

> Looks like sourceBlockTimestamp covers "Source Chain Transaction Time".

Agreed

> Do we need to add timestamps to all the other status changes, or are they inferable somehow already?

Thanks, that's a big gap in the existing functionality. I had planned to get that from other chains. I'll see about adding it here.